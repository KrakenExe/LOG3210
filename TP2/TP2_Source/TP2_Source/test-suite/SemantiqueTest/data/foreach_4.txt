Declaration
real a;
listreal [] e;
EndDeclaration

foreach(num d: e) {
    a = 1.0;
}