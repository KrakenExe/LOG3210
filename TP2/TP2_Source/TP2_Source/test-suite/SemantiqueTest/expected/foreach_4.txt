Array type listreal is incompatible with declared variable of type num...